src/lib/Scenes/Artwork/Components/__tests__/AuctionCountDownTimer-tests.tsx

```diff
@@ -1,33 +1,97 @@
 import { Theme, TimeRemaining } from "@artsy/palette"
 import { mount } from "enzyme"
 import { ArtworkFixture } from "lib/__fixtures__/ArtworkFixture"
-import { Settings } from "luxon"
+import { DateTime } from "luxon"
 import React from "react"
-import { AuctionCountDownTimer } from "../AuctionCountDownTimer"
-
-const realNow = Settings.now
-const realDefaultZone = Settings.defaultZoneName
+import { AuctionCountDownTimer, timeUntil } from "../AuctionCountDownTimer"
 
 describe("AuctionCountDownTimer", () => {
-  beforeAll(() => {
-    Settings.defaultZoneName = "America/New_York"
-    Settings.now = () => new Date("2019-08-15T12:00:00+00:00").valueOf()
-  })
+  const dateNow = 1565870400000 // 2019-08-15T12:00:00.000Z in milliseconds
+  let oneDayAgo
+  let oneDayFromNow
+  let oneYearAgo
+
+  beforeEach(() => {
+    jest.useFakeTimers()
+
+    // 2019-08-15T12:00:00.000Z
+    Date.now = () => dateNow
```

Comment:
@lilyfromseattle We need to reset this back to the real `Date.now` after the tests complete.